cdap-ui/app/cdap/components/FieldLevelLineage/v2/FllTable/index.tsx

```diff
@@ -50,16 +50,19 @@ const styles = (theme) => {
         paddingRight: '10px',
         borderTop: `1px solid ${theme.palette.grey[500]}`,
         height: '21px',
+        alignItems: 'center',
+        padding: '0',
+        gridTemplateColumns: '1fr auto !important',
       },
       ' & .grid-row:hover': {
         backgroundColor: theme.palette.grey[700],
-        cursor: 'pointer',
       },
       ' & .grid-row.selected': {
         backgroundColor: theme.palette.yellow[200],
         color: theme.palette.orange[50],
         fontWeight: 'bold',
         cursor: 'unset',
+        gridTemplateColumns: '1fr auto !important',
```

Comment:
is the `!important` necessary? I think seems like it's already set by grid row.

Reply:
If the !important isn't there, the columns get set by grid-row which doesn't quite work. Will add a comment along those lines.

Reply:
That said, this specific line is redundant, so removing